public/css/style.css

```diff
@@ -0,0 +1,655 @@
+@import url(https://fonts.googleapis.com/css?family=Dosis:400,600,700);
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box; }
+
+body {
+  font-family: 'Dosis', sans-serif;
+  font-size: 16px; }
+
+a {
+  text-decoration: none;
+  color: #000000; }
+
+img {
+  max-width: 100%; }
+
+.wrapper {
+  width: 80%;
+  margin: 0px auto; }
+
+.arrow {
+  border: solid #fff;
+  border-width: 0 2px 2px 0;
+  display: inline-block;
+  float: right;
+  padding: 6px; }
+
+header {
+  box-shadow: 0px 1px 11px #88888840; }
+  header .main_header {
+    display: flex;
+    padding: 0px;
+    min-height: 50px; }
+    header .main_header .main_header__logo {
+      flex: 1; }
+    header .main_header #m_menu {
+      display: none; }
+    header .main_header .main_header_logo {
+      flex-basis: 20%; }
+    header .main_header .nav {
+      flex-basis: 80%;
+      display: flex;
+      width: 100%;
+      position: relative; }
+    header .main_header ul li {
```

Comment:
Avoid nesting selectors more than 3 levels. 
You should be using BEM methodology to create selectors. https://css-tricks.com/bem-101/

Reply:
done 
i manage the nesting selection upto 3 levels